frontend/components/site/CommitSummary.jsx

```diff
@@ -1,7 +1,6 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 
-import api from '../../util/federalistApi';
```

Comment:
if the CommitSummary component no longer draws data from the API, the copmonent test doesn't have to mock it anymore. So you probably can cut a lot of the sinon stubs from that. You may want to move them into the siteBuildLogs.test.js, though?

Reply:
I actually think you could uncomment these, make sure to pass in build details in your new format, and actually test that commitsummary does commitsummary things

https://github.com/cloud-gov/pages-core/blob/8feae482078fe731a4aa8471251f4e5e0d732e12/test/frontend/components/site/CommitSummary.test.js#L47

Reply:
Very good point; that needs reorganizing.

Reply:
Whoops; I overlooked your third comment. Hold that thought.

Reply:
CommitSummary test improved. Still working on siteBuildLogs test.